.github/workflows/step_tests-conda.yml

```diff
@@ -38,29 +41,29 @@ jobs:
           use-only-tar-bz2: true
 
       - name: Install from source
-        shell: pwsh
         run: python -m pip install -e '.[test-external,test-cov]'
 
-      - name: Create kernel
-        shell: pwsh
-        run: |
-          python -m ipykernel install --name jupytext-ci --user
-
       - name: Conda list
-        shell: pwsh
         run: conda list
 
+      - name: Install a Jupyter Kernel
+        run: python -m ipykernel install --name jupytext-ci --user
+
       - name: Test lab extension
-        shell: pwsh
         run: |
           # Check extension
           jupyter labextension list
           jupyter labextension list 2>&1 | grep -ie "jupyterlab-jupytext.*OK"
           python -m jupyterlab.browser_check
 
       - name: Test with pytest
-        shell: pwsh
-        run: pytest --cov
+        # We use an explicit coverage file otherwise the
+        # codecov action below does not find it
+        run: pytest --cov . --cov-report xml:coverage.xml
 
       - name: Upload coverage
         uses: codecov/codecov-action@v3
+        with:
+          fail_ci_if_error: true
```

Comment:
`fail_ci_if_error` has no effect here because this step is not triggered if the test step failed. You can add `if: always()` if you do want it to upload failed code coverage.